app.js

```diff
@@ -47,46 +50,86 @@ var answersRight = [
+      pAns.className = 'incorrect';
+      pAns.textContent = ansWrong;
+      image.src = './img/carl_wrong.jpg';
+      }
+    } else {
+      var left = 2;
+      pQues.textContent = ques;
+
+      while (parseInt(userInput) !== ans) {
+        if (left === 0) {
+          break;
+        }
+        if (parseInt(userInput) < ans) {
+          left--;
+          pAns.textContent = ansWrong[0];
+          pAns.className = 'incorrect';
+          image.src = './img/carl_wrong.jpg';
+          userInput = parseInt(prompt(ansWrong[0] + ' ' + ques + ' You have ' + left + ' tries left.'));
+          console.log('This answer is incorrect. The counter is currently ' + counter);
+        } else if (parseInt(userInput) > ans) {
+          left--;
+          pAns.textContent = ansWrong[1];
+          pAns.className = 'incorrect';
+          image.src = './img/carl_wrong.jpg';
+          userInput = parseInt(prompt(ansWrong[1] + ' ' + ques + ' You have ' + left + ' tries left.'));
+          console.log('This answer is incorrect. The counter is currently ' + counter);
+        }
+      }
+      if (parseInt(userInput) === ans) {
+        counter++;
+        console.log(counter);
+        pAns.textContent = ansRight;
+        pAns.className = 'correct';
+        image.src = './img/carl_right.jpg';
+        console.log('This answer is correct. The counter is currently ' + counter);
+
       }
     }
-    counter++;
-    pqEl.textContent = 'Question: ' + questions;
-    paEl.textContent = 'Correct Answer (' + answers + '): ' + answersRight +'. You\'ve gotten ' + counter + ' correct! Good job!';
-  } else if (userInput.toLowerCase() === answers[0] || userInput.toLowerCase() === answers[1]) {
-    counter++;
-    pqEl.textContent = 'Question: ' + questions;
-    paEl.textContent = 'Correct Answer (' + answers + '): ' + answersRight +'. You\'ve gotten ' + counter + ' correct! Good job!';
+    console.log(counter);
+  if(counter === 5) {
+  document.getElementById('finalCount').textContent = 'You got ' + counter + ' right! It\'s kind of creepy that you know me this well. Should I be worried?';
+  } else if (counter <= 4 && counter >= 2) {
+    document.getElementById('finalCount').textContent = 'You got ' + counter + ' right! You know me as well as some of my best friends!';
   } else {
-    pqEl.textContent = 'Question: ' + questions;
-    paEl.textContent = 'Incorrect Answer (The correct answer was: (' + answers + ')): ' + answersWrong;
+    document.getElementById('finalCount').textContent = 'You got ' + counter + ' right! You don\'t know me at all!';
   }
 }
 
-//This section contains the for loop that iterates the function calls to begin the guessing game.
-for (i = 0; i <= 4; i++) {
-    game(questions[i], answers[i], answersRight[i], answersWrong[i], pqEl[i], paEl[i]);
+for(var i = 0; i <= 2; i++) {
+  game(questions[i], answers[i], answersRight[i], answersWrong[i], pq[i], pa[i], images[i]);
+};
+
+for(var i = 3; i < 5; i++) {
```

Comment:
I used two for loops because I couldn't figure out how to type check the user input for strings and numbers without copying Scott's code exactly. Not a big fan of copying the code from the instructor.